pytest_client_tools/insights_client.py

```diff
@@ -237,6 +237,20 @@ def core_version(self):
         assert m
         return Version(m.group(1))
 
+    @property
+    def client_version(self):
+        """
+        Return the version of insights-client as
+        [`Version`][pytest_client_tools.util.Version] object.
+
+        :return: The version of the insights-client in use.
+        :rtype: pytest_client_tools.util.Version
+        """
+        proc = self.run("--version")
+        m = re.search(r"^Client: (.+)+$", proc.stdout, re.MULTILINE)
```

Comment:
there is a stray `+` after the capture group; while the regex seems valid (to my surprise), I'd avoid adding bits that are not exactly clear